cmd/thanos/config.go

```diff
@@ -133,6 +134,8 @@ func (sc *shipperConfig) registerFlag(cmd extkingpin.FlagClause) *shipperConfig
 	cmd.Flag("shipper.upload-compacted",
 		"If true shipper will try to upload compacted blocks as well. Useful for migration purposes. Works only if compaction is disabled on Prometheus. Do it once and then disable the flag when done.").
 		Default("false").BoolVar(&sc.uploadCompacted)
+	cmd.Flag("shipper.upload-debug-meta-files", "If true shipper will upload debug meta files which can be useful for debugging.").
+		Default("false").BoolVar(&sc.uploadDebubgMetaFiles)
```

Comment:
We are using the value `true` by default in a lot of places (tests etc) so why the default is "false"? IMO it makes more sense to have the default as "true" as it will maintain backward compatibility as well.